vault/resource_mount.go

```diff
@@ -92,6 +92,16 @@ func mountResource() *schema.Resource {
 				Computed:    true,
 				Description: "Enable seal wrapping for the mount, causing values stored by the mount to be wrapped by the seal's encryption capability",
 			},
+
+			"external_entropy_access": {
+				Type:        schema.TypeBool,
+				Required:    false,
+				Optional:    true,
+				Default:     false,
+				ForceNew:    true,
+				Computed:    false,
```

Comment:
We should be able to omit `Required` and `Computed` here, they both will receive their default value of  `false`.

Reply:
Sounds good. I've removed them in fb4facc.